india_compliance/gst_india/utils/test_e_invoice.py

```diff
@@ -45,8 +48,116 @@ def setUpClass(cls):
         )
         update_dates_for_test_data(cls.e_invoice_test_data)
 
-    @change_settings("Selling Settings", {"allow_multiple_items": 1})
     def test_get_data(self):
```

Comment:
better function name

Reply:
`test_e_invoice_request_data` would be fine?